config/300-eventing.yaml

```diff
@@ -57,6 +57,9 @@ spec:
         spec:
           description: Spec defines the desired state of KnativeEventing
           properties:
+            version:
```

Comment:
that;s an interesting approach to specify the actual version here, nice

I wonder what if the _current_ is not directly compatible ? e.g. move from 0.14 -> 0.17  - or out of scope ? (totally fine, just wondering)

Reply:
I think the current plan is to prevent it, i.e. toss an error if spec.version is plus-or-minus 0.1 from status.version.